content/user/payment/paypal_ipn.md

```diff
@@ -5,14 +5,33 @@ category: payment
 weight: 10
 ---
 
-## Things to check if PayPal payments are not working
+These are things to check if PayPal payments are not working.
 
+---
 ### Are you getting specific error messages?
 See the [PayPal Errors](/user/payment/paypal_errors/) page to investigate.
 
+---
+### Are you getting "Whoops?" timeouts when you return to your cart after payment? 
+
+Note: This only applies to PayPal Standard. 
```

Comment:
Insomuch as this document speaks about PayPal modules, this "only applies to Standard" comment is accurate.

However, as far as this samesite cookie issue in general, it applies to (likely) all payment modules which host payment on an external site that the customer's browser gets redirected to for payment and then must return back to the store for order completion.
ePath is another one for which this is a known requirement.
(An exception would be where the payment module receives the session id as part of its transaction request and returns it again when redirecting the customer back to the store ... but most gateways don't like to cooperate that intelligently, so this ideal situation is unfortunately rare.)

Reply:
Understood - but did you want to create a whole new "payment modules troubleshooting" doc for this issue?

Reply:
Not at this stage.